gobdd.go

```diff
@@ -91,8 +92,17 @@ type stepDef struct {
 	f    interface{}
 }
 
+type tester interface {
```

Comment:
This type should be exported, since it's used in an exported function. I would probably keep calling it `T` or `TestingT` (as seen in testify).

Reply:
good point! changed 👍